packages/phone-number-privacy/common/src/interfaces/requests.ts

```diff
@@ -15,14 +15,24 @@ import {
   SequentialDelayDomain,
 } from '../domains'
 
-export enum Endpoints {
+export enum CombinerEndpoints {
```

Comment:
Nit: I'd suggest dropping the `s` on this type and the one below. Types including enums are generally not plural. (i.e. `endpoint` is of type `Endpoint`. `Endpoints` might confuse readers to think its an array)
```suggestion
export enum CombinerEndpoint {
```